module/Motion/CorrectingRotation.h

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file   CorrectingRotation.h
+ * @brief  角度補正回頭
+ * @author aridome222
+ */
+
+#ifndef CORRECTING_ROTATION_H
+#define CORRECTING_ROTATION_H
+
+#include "Motion.h"
+#include "Mileage.h"
+#include "Timer.h"
+#include "SpeedCalculator.h"
+#include "StringOperator.h"
+#include "SystemInfo.h"
+
```

Comment:
```suggestion
#include "CompositeMotion.h"
```
CompositeMotion 継承とのことなので、include も。